research/activity29-battleship.yaml

```
@@ -187,21 +187,64 @@ sections:
-          4. CRITICAL: If user_sunk_ship_this_round contains a ship name, celebrate the player destroying the AI ship in 2 sentences describing the carnage at sea
-          5. CRITICAL: If game_over is true, announce the victory:
-             - If user_wins is true: Celebrate the player's total victory with excitement!
-             - If ai_wins is true: Express dismay at the player's defeat!
-
-          Describe the sights and sounds of naval warfare! You are the game system rooting for the player!
+          You are a battleship narrator. Each prompt has its own specific role - follow the individual prompt instructions precisely.
+        feedback_prompts:
+          - name: "Shot Report"
+            tokens_for_ai: |
+              🎯 Report ONLY the hit/miss results for both shots this turn. DO NOT report ship sinking or game over.
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)